src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas2/Cas2AutoScript.kt

```diff
@@ -71,6 +71,7 @@ class Cas2AutoScript(
         submittedAt = submittedAt,
         schemaVersion = jsonSchemaService.getNewestSchema(Cas2ApplicationJsonSchemaEntity::class.java),
         schemaUpToDate = true,
+        telephoneNumber = "123 456 789",
```

Comment:
hmm, I see that this is necessary and works fine. 

But I wonder if we should make an effort to keep the seeded data realistic: as is, this change would add the `telephoneNumber` property to unsubmitted applications as well as to submitted ones. 

If you see https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/9aa9062b07ee4f76b780f0e53891132dd0a16191/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas2/Cas2AutoScript.kt#L81 from my open PR https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1320, we could introduce something like `setFirstClassProperties()` to just affect the applications which are in either the `SUBMITTED` or `IN_REVIEW` states?